### PR TITLE
1319: useNotification

### PR DESCRIPTION
**Pull request by AugusteBa**
Refers to #1319 
We will need to make a tiny change once my PR gets merged into `openfin-react-hooks`, but for now - this works.

### CODE REVIEW

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js [hunk 1]
```diff
@@ -36,17 +43,16 @@ const Watchlist = () => {
...
-        watchlistName: 'My Watchlist',
-        alreadyInWatchlist,
-        messageText: `${alreadyInWatchlist ? ' moved' : ' added'} to the top`
-      }
-    });
+    setNewSymbol(newSymbol);
+    if (notification) {
+      notification.launch({
+        url: 'notification.html',
+        timeout: 'never'
```
Comment (kangelSL) [inline packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: The notification is currently appearing and never vanishing; should it be made to disappear after a programmed interval? Currently the user needs to drag it away to make it disappear which is not immediately obvious; is there a way to add an 'X' to show the user how they can close it?

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js [hunk 2]
```diff
@@ -59,6 +65,33 @@ const Watchlist = () => {
...
+            />
+          </div>
+          <div id="notification-content">
+            <p id="title">WATCHLIST UPDATED</p>
+            <hr />
+            <p id="info">
+              <span id="symbol">{newSymbol}</span>
+              <span id="message">{`${
+                alreadyInWatchlist ? ' moved' : ' added'
+              } to the top`}</span>
```
Comment (kangelSL) [inline packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: If I add something new to the watchlist, I get the message 'moved to the top', even if the list is empty. Should it be 'added to the top'? Also when I add the same item twice the notification fires twice. Is this the correct behaviour?

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js (hunk 1 above)
Comment (AugusteBa) [reply packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Thanks, Katriona, that's a very good point, removed it now altogether, which defaults to 3600 milliseconds. 
There is a way to add X button, but there's no such requirement in UX specs.

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js (hunk 2 above)
Comment (AugusteBa) [reply packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Yeah, that is expected behavior (firing twice). Messages are going to change either way in the Notification UX refactor PR.

**Pull request merged**